.github/workflows/release-packages.yaml

```diff
@@ -58,6 +58,7 @@ jobs:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.DB_CI_CPROVER_ACCESS_TOKEN }}
 
   windows-msi-package:
+    runs-on: windows-2019
```

Comment:
What is `windows-2019`? I know of `vs2019` but what is that tag?

Reply:
@NlightNFotis it's short for Windows Server 2019, see https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners